.circleci/config.yml

```diff
@@ -4,12 +4,9 @@
 # Main Config
 version: 2
 
-base_image: &base_image
-  image: circleci/node:carbon
-
 job_common: &job_common
   docker:
-    - <<: *base_image
+    - image: circleci/node:carbon-browsers
```

Comment:
What is the benefit of moving this out of the base image? Also, what does the `-browsers` tag add?

Reply:
Having the base image in its own "variable" seemed like an unnecessary abstraction.
The browsers tag was a configuration that I was originally going to use. It provides a `Selenium Webdriver` service and I believe a headless chrome instance with it within the docker container. But I eventually decided that going with SauceLabs would be more beneficial rather than going with a local instance in a docker container I can't truly diagnose if there are issues.

https://circleci.com/docs/2.0/browser-testing/#selenium